utils/src/test/kotlin/ArchiveUtilsTest.kt

```diff
@@ -76,5 +77,43 @@ class ArchiveUtilsTest : StringSpec() {
             fileB.exists() shouldBe true
             fileB.readText() shouldBe "b\n"
         }
+
+        "Debian deb archive can be unpacked" {
+            val archive = File("src/test/assets/testpkg.deb")
+
+            archive.unpack(outputDir)
+
+            val extractedScriptFile = outputDir.resolve("usr/bin/test")
+            extractedScriptFile.exists() shouldBe true
```

Comment:
Please use matchers like `shouldBe aFile()` which provide nicer output on failure that just a mismatch to "true".